sirepo/pkcli/db.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+u"""Database utilities
+
+:copyright: Copyright (c) 2017 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+
+
+def upgrade():
+    """Upgrade the database"""
+    from pykern import pkio
+    from sirepo import simulation_db
+    from sirepo import server
+
+    server.init()
+    to_rename = []
```

Comment:
Nope. Good catch. :)

I'll leave it in, because it's harmless, and the upgrade is ephemeral.